src/filter.js

```diff
@@ -42,7 +42,21 @@ const EXCLUDED_MATERIAL_TYPES = [
 
 export default function (record, earliestCatalogTime) {
 	const leader = record.varFields.find(f => f.fieldTag === '_');
-	const materialType = record.materialType.code.trim();
+
+	const materialType = function () {
+		if (record.materialType === undefined) {
+			return false;
+		}
+
+		if (record.materialType && record.materialType.code) {
+			let result = record.materialType.code.trim();
+			return result;
```

Comment:
We don't use `let` unless really needed. Just do `return record.materialType.code.trim();`